CHANGES.md

```diff
@@ -847,6 +870,13 @@ OpenSSL 3.0
    time.  Instead applications should use L<EVP_DigestSignInit_ex(3)>,
    L<EVP_DigestSignUpdate(3)> and L<EVP_DigestSignFinal(3)>.
 
+   These low level DSA functions have been deprecated without replacement:
+
+   DSA_clear_flags, DSA_dup_DH, DSAparams_dup, DSA_set_flags and
+   DSA_test_flags.
+
+   The DSA_FLAG_CACHE_MONT_P flag has been deprecated without replacement.
+
    Finaly functions that assign or obtain DSA objects from an EVP_PKEY such as
```

Comment:
While you are here - change this to 'finally'

Reply:
Done, and a second instance of the same.